svd2py/objects.py

```diff
@@ -21,7 +20,13 @@ class SvdChildElement:
 
 class SvdTypeParser:
     def _get_bool(self, value: str) -> bool:
-        return bool(strtobool(value))
+        value = value.lower()
+        if value in ("true", "1"):
+            return True
+        elif value in ("false", "0"):
+            return False
+        else:
+            raise ValueError(f"Invalid boolean value {val}")
```

Comment:
I guess it is a typo. Should be `{value}`.

Reply:
Oops! Should be resolved now :)